build/wikia/transformers/transformWarframe.mjs

```diff
@@ -28,7 +27,7 @@ const mapColors = async (oldFrame, imageUrl) => {
  * @param {Record<string, Object>} blueprints blueprint objects
  * @returns {Promise<WikiaWarframe>}
  */
-module.exports = async (oldFrame, imageUrls, blueprints) => {
+export async function transformWarframe(oldFrame, imageUrls, blueprints) {
```

Comment:
as elsewhere, don't use functions, use named closures, especially since we're exporting named anyway